auditorium/index.html

```diff
@@ -5,10 +5,9 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <base href="/auditorium/">
-    <link rel="stylesheet" type="text/css" href="/normalize.css">
-    <link rel="stylesheet" type="text/css" href="/skeleton.css">
+    <link rel="stylesheet" type="text/css" href="/tachyons.css">
   </head>
-  <body>
+  <body style="background-color: #fdfdfd;">
```

Comment:
Is there a Tachyons class that would do the same thing?

Reply:
unfortunately not - just too bright for standard tachyons colors - but i will change the colorways anyway because of the audit results in term of contrast/readability